src/TipBox.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-    <div class="alert container" :class="[boxStyle, addClass, lightStyle, seamlessStyle]" :style="customStyle">
-        <div v-if="!isDefault" class="icon-wrapper" :class="[iconStyle]">
```

Comment:
Previously, the code renders the `icon-wrapper` div if the type of box is not a default box. However, I change this because the type of the box can **not** be default, and still have an icon (inserted via `icon="..."`).

Example:
```
<box icon=":rocket:">
    Lorem ipsum dolor sit amet,
</box>
```
![image](https://user-images.githubusercontent.com/27397021/74598277-cc917000-50a9-11ea-8ee7-1e6c28d581d7.png)